tests/foreman/cli/test_variables.py

```diff
@@ -16,122 +16,185 @@
 @Upstream: No
 """
 
+from robottelo import ssh
+from robottelo.cli.base import CLIReturnCodeError
+from robottelo.cli.environment import Environment
+from robottelo.cli.factory import make_hostgroup, make_variable
+from robottelo.cli.host import Host
+from robottelo.cli.hostgroup import HostGroup
+from robottelo.cli.proxy import Proxy
+from robottelo.cli.puppet import Puppet
+from robottelo.cli.variable import SmartVariable
```

Comment:
Module name `variable`, class and helpers name `smart[_\- ]?variable`... How about making all of them consistent?